src/main/java/de/tum/in/www1/artemis/domain/AuxiliaryRepository.java

```diff
@@ -157,7 +156,7 @@ public VcsRepositoryUrl getVcsRepositoryUrl() {
             return new VcsRepositoryUrl(repositoryUrl);
         }
         catch (MalformedURLException e) {
-            LoggerFactory.getLogger(AuxiliaryRepository.class).error("Malformed URL " + getRepositoryUrl() + " you could not be used to instantiate VcsRepositoryUrl.", e);
+            e.printStackTrace();
```

Comment:
Please log that properly, the logging was good, my suggestion was just to make the logger constant.

Reply:
Fixed